notification/check/check.go

```diff
@@ -214,7 +214,8 @@ func UnmarshalJSON(b []byte) (influxdb.Check, error) {
 	}
 	if err := json.Unmarshal(b, &raw); err != nil {
 		return nil, &influxdb.Error{
-			Msg: "unable to detect the check type from json",
+			Code: influxdb.EInvalid,
```

Comment:
useful messaging for downstream consumers, aka pkger looks wants to know that it failed b/c it is invalid, meaning, it is a check we can skip